Gordon360/Services/RecIM/ParticipantService.cs

```diff
@@ -140,39 +140,34 @@ public async Task PostParticipant(int participantID)
                 //No defined end date for creation
             });
             await _context.SaveChangesAsync();
+            var username = GetAccountByParticipantID(participantID).AD_Username;
+            var participant = GetParticipantByUsername(username);
+            return participant;
         }
-        public async Task UpdateParticipant(ParticipantPatchViewModel updatedParticipant)
+        public async Task<ParticipantActivityCreatedViewModel> UpdateParticipantActivity(string username, ParticipantActivityPatchViewModel updatedParticipant)
         {
-            int participantID = Int32.Parse(_accountService.GetAccountByUsername(updatedParticipant.Username).GordonID);
-            if (updatedParticipant.ActivityID is not null)
-            {
-                var participantActivity = _context.ParticipantActivity
-                                            .FirstOrDefault(pa => pa.ParticipantID == participantID 
-                                                && pa.ActivityID == updatedParticipant.ActivityID);
-                var priv = _context.PrivType
-                                .FirstOrDefault(pt => pt.Description == updatedParticipant.ActivityPrivType);
+            int participantID = Int32.Parse(_accountService.GetAccountByUsername(username).GordonID);
```

Comment:
If you do not like using the `Int32.Parse(...` everywhere, perhaps it would be a good idea to create a helper method for this as well. i.e. `int ParticipantService.GetParticipantIDByUsername(string username)`

Reply:
same comment as above, we'll either be moving things into views and/or using ad_username as a unique identifier